supervisor/loggers.py

```diff
@@ -406,6 +406,8 @@ def handle_syslog(logger, fmt):
     handler.setLevel(logger.level)
     logger.addHandler(handler)
 
+    return logger
+
```

Comment:
It is weird that some of these `handle_` functions return a `logger` and some don't.  Why was this change made?

Reply:
I made this change because I needed a logger at dispatchers.py:154. Maybe loggers are only returned from the handle_ functions when they're needed?